src/test/java/de/tudarmstadt/ukp/wikipedia/parser/ParserTest.java

```diff
@@ -161,6 +164,39 @@ public void testExtractingLinksWithColons(){
         testAnchorsInText(pp);
     }
 
+	@Test
+	public void testExtractingLinksOtherLang(){
+
+		String text = "* La Douceur de croire, pièce en trois actes, Théâtre Français, 8 July 1899\n" +
+				"\n" +
+				"[[:France]]In collaboration with [[:fr:André Delavigne]] " +
+				"* Blakson père [[::::::Potato|Pommes]] et fils,[[fr:Something]] comédie en quatre actes, Théâtre de l'Odéon\n" +
+				"* Les petites  marmites, comédie en trois actes, Théâtre du Gymnase\n" +
+				"[[cite:Gundam]] * Voilà Monsieur !, comédie en un acte, Théâtre du Gymnase";
+
+		MediaWikiParserFactory pf = new MediaWikiParserFactory(WikiConstants.Language.english);
+		MediaWikiParser parser = pf.createParser();
+
+		ParsedPage pp = parser.parse(text);
+
+		List<String> uris = getUrisInParagraphs(pp);
+		assertFalse(uris.contains("André_Delavigne"));
```

Comment:
Nope, It won't contain them.

If an annotation comes as `[[LANGUAGE:XX]]` or `[[::::LANGUAGE:XX]]`  then it wont end up in the field that we use for the paragraphLinks.